### ISSUE TITLE
Support: Ignore or Skip previous Episodes?

### ISSUE THREAD

**Issue by annomatik** [pymedusa/Medusa#4881]
**Describe the bug**
Not a bug, but a support request.

I "killed" 20 Spongebob episodes by telling Medusa to replace lower quality episodes with 1080p ones. The problem: Apparently, the Spongebob episode list on thetvdb.com is not used by all the scene guys. Instead of Season 11, the episodes downloaded where classified as Season 10, thus the Season 11 episodes overwrote my Season 10 ones. Luckily, I had a backup.

Long story short: How can I tell Medusa to only consider new stuff and leave the old stuff alone? Is it "Ignore previous episodes" or "Skip previous episodes" and what's the difference?

Thanks!

**Comment by medariox** [comment]
Both would work. Skipped shows the episode as missing while ignored doesn't. That's the only difference.

**Comment by annomatik** [comment]
Perfect, thanks!

Thread can be closed.
